library/std/src/android-api.c

```diff
@@ -0,0 +1,4 @@
+// Used from backtrace-rs' build script to detect the value of the `__ANDROID_API__`
+// builtin #define
+
+APIVERSION __ANDROID_API__
```

Comment:
Seems a little messy to have to duplicate this file. Can we instead modify `backtrace-rs/build.rs` to make it directory-independent somehow?

Maybe change it to have `pub fn main` instead and then it can do
```rust
let android_api_c = std::path::Path::new(std::file!()).parent().unwrap().join("src/android-api.c");
let expansion = match cc::Build::new().file(android_api_c).try_expand() {
// ...
```